src/createBaseForm.js

```diff
@@ -45,9 +45,9 @@ function createBaseForm(option = {}, mixins = []) {
         };
       },
 
-      componentWillReceiveProps(nextProps) {
+      componentWillReceiveProps(nextProps, nextContext) {
         if (mapPropsToFields) {
-          this.fields = mapPropsToFields(nextProps);
+          this.fields = mapPropsToFields(nextProps, nextContext);
```

Comment:
Sorry, I make a mistake.

It seems that you cannot get any value from `nextContext`, for this HOC doesn't subscribe any value, right?